<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testvector</class>
 <widget class="QWidget" name="testvector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Uji Vektor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Uji Vektor ChaCha20</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="2,5">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Masukan HEX</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Key :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="txtKey">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Nonce :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="txtNonce">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Block Counter :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="txtbc">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Plaintext :</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPlainTextEdit" name="txtPlaintext"/>
            </item>
            <item row="4" column="1">
             <widget class="QWidget" name="widget" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="20,1">
               <property name="spacing">
                <number>1</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="btnTest">
                 <property name="minimumSize">
                  <size>
                   <width>20</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Test</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnClear">
                 <property name="minimumSize">
                  <size>
                   <width>5</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>X</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Keluaran HEX</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="txtCipherText">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
